internal/distro/rhel8/azure.go

```diff
@@ -93,6 +93,14 @@ func azureImgType() imageType {
 	}
 }
 
+func azureEAPImgType() imageType {
```

Comment:
nitpick: to be consistent, we should consider naming this image `azure-eap-rhui`, because I would expect that we may end up building the same image as BYOS in the service or on premise 🤔

Reply:
Could do.